App.js

```diff
@@ -1,19 +1,24 @@
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, View } from 'react-native';
+import Amplify from 'aws-amplify'
+import awsconfig from './src/aws-exports'
+import Home from './src/Home';
+import Header from './src/Header';
 
+Amplify.configure(awsconfig)
 export default function App() {
   return (
     <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
+      <Header />
+      <Home />
       <StatusBar style="auto" />
```

Comment:
Should `StatusBar` be above the other components?

Reply:
Nice Catch!. I'll update that.